llama_index/indices/knowledge_graph/retrievers.py

```diff
@@ -274,9 +274,14 @@ def _retrieve(
             logger.info("> No relationships found, returning nodes found by keywords.")
             if len(sorted_nodes_with_scores) == 0:
                 logger.info("> No nodes found by keywords, returning empty response.")
-            return [
-                NodeWithScore(node=TextNode(text="No relationships found."), score=1.0)
-            ]
+                return [
+                        NodeWithScore(node=TextNode(text="No relationships found."), score=1.0)
+                    ]
+            
+            # In else case the sorted_nodes_with_scores is not empty 
+            # thus returning the nodes found by keywords
+            logger.info("> No relationships found by embeddings, returning nodes found by keywords.")
```

Comment:
I think this logger is actually redundant since we already have a log statement in L. 276.

Reply:
Yes, makes sense. 
Quick check: Is this the one causing lint error?